exonum-java-binding-common/src/main/java/com/exonum/binding/common/serialization/UtcZonedDateTimeSerializer.java

```diff
@@ -18,23 +18,25 @@
 package com.exonum.binding.common.serialization;
 
 import static com.exonum.binding.common.serialization.SerializationUtils.checkLength;
+import static com.google.common.base.Preconditions.checkArgument;
 
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
 import java.time.Instant;
 import java.time.ZoneOffset;
 import java.time.ZonedDateTime;
 
-enum ZonedDateTimeSerializer implements Serializer<ZonedDateTime> {
+enum UtcZonedDateTimeSerializer implements Serializer<ZonedDateTime> {
   INSTANCE;
 
-  private static final int BYTE_BUFFER_CAPACITY = Long.BYTES + Integer.BYTES;
+  private static final int SERIALIZED_ZDT_SIZE = Long.BYTES + Integer.BYTES;
 
   @Override
   public byte[] toBytes(ZonedDateTime value) {
+    checkArgument(value.getZone() == ZoneOffset.UTC, "Serialized ZonedDateTime value should be in UTC");
```

Comment:
1. Please add the actual value
2. Remove "Serialized" as the value is just an object.